src/IndexedRecord.php

```diff
@@ -2,11 +2,18 @@
 namespace Swis\LaravelFulltext;
 
 use Illuminate\Database\Eloquent\Model;
+use Illuminate\Support\Facades\Config;
 
 class IndexedRecord extends Model
 {
     protected $table = 'laravel_fulltext';
 
+    public function __construct()
+    {
+        parent::__construct();
+        $this->connection = Config::get('laravel-fulltext.db_connection', config('database.default'));
```

Comment:
This project uses the config helper instead of the facade. I prefer to keep it consistent, certainly in one line.